### PR TITLE
MDAnalysis GSoC proposal

### PR DESCRIPTION
**Pull request by ayushsuhane**
Hi

Thanks that you have chosen to apply for a NumFOCUS project. You can use pull
requests on this repository to work on a draft of your proposal before the
student application period begins to get early feedback.

Before you open a pull request with a draft of your proposal please make sure
that you have contacted your organization beforehand. Some organization prefer
if you communicate on their preferred channel (email/irc) and are more likely to
give you feedback then. On the [README][readme] we list the preferred contact
information of every organization and project idea.

best NumFOCUS GSoC Admins

[readme]: https://github.com/numfocus/gsoc/blob/master/README.md

### CODE REVIEW

**Comment by ayushsuhane** [comment]
Hi, I am sorry, I have been very busy lately. I understand all the developers are more busy than me, but I would appreciate if anyone can give me the comments to edit the proposal. I have shared an incomplete draft. I have yet to write the pseudo-codes and files which I have to modify. I have written the outline in a crude sense. Kindly edit/provide suggestions, I will surely incorporate and submit my application before the deadline.

**Comment by kain88-de** [comment]
Hi, @ayushsuhane thanks for applying.

please upload the draft also to the summer of code site asap to make sure your application is valid. You will have to set it to finalized by the Tuesday evening to have a valid application for GSoC. Google only counts applications on their own page. Google will want a final PDF so you can either convert the md document and link to it or use the google docs that they offer by default.

**Comment by kain88-de** [comment]
ping @richardjgowers @jbarnoud

File: Ayush.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+### **Phase 2**
+(June 26 - August 25)
+Implement OCTree for the required case of neighbour searching problem. Produce benchmark cases for periodic and non periodic system for the memory usage and time to build the trees
+Finalize the method for both the use cases for MDAnalysis users. 
+Submit a PR to implement another keyword in selections.py alogn with docs and tests
+
+### **Final Week**
+Submit another PR to provide Use case for contact maps and RDF using the implemented method .
+At this stage you should finish up your project. At this stage you should make
```
Comment (jbarnoud) [inline Ayush.md]: Do not forget do remove the placeholder text.

**Comment by ayushsuhane** [comment]
Hi, I have uploaded the Google doc to summer of code page for comments. Thanks for the response. 

While most of the work I have written is based on benchmarks, I was wondering whether there are any specific method I should add in the proposal.

**Comment by ayushsuhane** [comment]
@kain88-de  @richardjgowers @jbarnoud  I have modified the proposal and uploaded on summer of code website. I have edited it in a google doc. If possible, please provide comments.

**Pull request closed without merging**